LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,19 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail Log Aliases:
+alias tailauto='tail -f /var/www/miq/vmdb/log/automation.log'
+alias tailevm='tail -f /var/www/miq/vmdb/log/evm.log'
+alias tailaws='tail -f /var/www/miq/vmdb/log/aws.log'
+alias tailfog='tail -f /var/www/miq/vmdb/log/fog.log'
+alias tailrhevm='tail -f /var/www/miq/vmdb/log/rhevm.log'
+alias tailprod='tail -f /var/www/miq/vmdb/log/production.log'
+alias tailpolicy='tail -f /var/www/miq/vmdb/log/policy.log'
+alias tailpglog='tail -f /var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/postgresql.log'
```

Comment:
That's a great idea and I have updated accordingly.  

We can not use the $vmdb variable though as it also includes the "cd" command.